lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -648,6 +659,24 @@ class DropDown {
     this._menuEl = this._dom.find('.lh-tools__dropdown', this._dom.document());
     this._menuEl.addEventListener('keydown', this.onMenuKeydown);
     this._menuEl.addEventListener('click', menuClickHandler);
+
+    // i18n dropdown menu
+    this._dom.find('#lh-dropdown__print-summary', this._menuEl).textContent =
```

Comment:
could we adopt a pattern of giving these elements something like a `data-i18n` attribute - we could then iterate over all such elements and lookup the provided variable in `Util.UIStrings`.

Reply:
Done, good idea!